crates/sui-graphql-rpc/src/types/coin_metadata.rs

```diff
@@ -9,6 +9,7 @@ use async_graphql::*;
 
 use sui_types::coin::CoinMetadata as NativeCoinMetadata;
 
+/// The metadata for an object of type Coin.
 pub(crate) struct CoinMetadata {
     pub super_: MoveObject,
```

Comment:
could benefit from a comment that this is the graphql::MoveObject (but can also be seen from rust-analyzer)